src/apps/dictionaries/components/DictionaryForm.tsx

```diff
@@ -290,8 +292,6 @@ const DictionaryForm: React.FC<Props> = ({
               <Field
                 multiple
                 fullWidth
-                multiline
-                rowsMax={4}
```

Comment:
same thing can we remove from source for if applicable

Reply:
This properties where not added in source.

Reply:
These properties were not added in source form. These were present only in dictionary form.